presto-docs/src/main/sphinx/release/release-332.rst

```diff
@@ -0,0 +1,84 @@
+=========================
+Release 332 (XX Apr 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix LDAP authentication not working if user belongs to multiple groups. (:issue:`3206`)
+* Fix query failure during planing phase for certain queries involving multiple joins. (:issue:`3149`)
+* Fix execution failure for queries involving huge IN on long decimal columns. (:issue:`3191`)
```

Comment:
It would be "decimal with precision larger than 18". I'll rephrase as:

`Fix execution failure for queries involving large ``IN`` predicates on decimal values with precision larger than 18.`